odoo/addons/base/models/res_lang.py

```diff
@@ -332,6 +332,17 @@ def format(self, percent, value, grouping=False, monetary=False):
 
         return formatted
 
+    def copy_data(self, default=None):
+        default = dict(default or {})
+
+        copy_header = _("%(header)s (copy)")
+        if "name" not in default:
+            default["name"] = copy_header % {"header": self.name}
+        if "code" not in default:
+            default["code"] = copy_header % {"header": self.code}
+        if "url_code" not in default:
+            default["url_code"] = copy_header % {"header": self.url_code}
```

Comment:
```suggestion
        if "name" not in default:
            default["name"] = _("%(header)s (copy)", header=self.name)
        if "code" not in default:
            default["code"] = _("%(header)s (copy)", header=self.code)
        if "url_code" not in default:
            default["url_code"] = _("%(header)s (copy)", header=self.url_code)
```

Reply:
This allows benefiting from the translation's fallback mechanism when there is a variable missing in the translation.